docs/data/material/customization/how-to-customize/how-to-customize.md

```diff
@@ -5,61 +5,59 @@ components: GlobalStyles
 
-As components can be used in different contexts, there are several approaches to customizing them. Going from the narrowest use-case to the broadest, these are:
+Material UI provides several different ways to customize a component's styles. Your specific context will determine which one is ideal. From narrowest to broadest use case, here are the options:
 
 1. [One-off customization](#1-one-off-customization)
 1. [Reusable style overrides](#2-reusable-style-overrides)
-1. [Dynamic variation](#3-dynamic-variation)
 1. [Global theme variation](#4-global-theme-variation)
 1. [Global CSS override](#5-global-css-override)
 
 ## 1. One-off customization
 
-You might need to change the style of a component for a specific implementation, for which you have the following solutions available:
+To change the styles of _one single instance_ of a component, you can use one of the following options:
 
-### Use the `sx` prop
+### The `sx` prop
 
-The easiest way to add style overrides for a one-off situation is to use the [`sx` prop](/system/basics/#the-sx-prop) available on all MUI components.
-Here is an example:
+The [`sx` prop](/system/basics/#the-sx-prop) is the best option for adding style overrides to a single instance of a component in most cases.
+It can be used with all Material UI components.
 
-{{"demo": "SxProp.js"}}
+{{"demo": "SxProp.js" }}
 
-Next you'll see how you can use global class selectors for accessing slots inside the component. You'll also learn how to easily identify the classes which are available to you for each of the states and slots in the component.
+#### Overriding nested component styles
 
-### Overriding nested component styles
+To customize a specific part of a component, you can use the class name provided by Material UI inside the `sx` prop.
 
-You can use the browser dev tools to identify the slot for the component you want to override. It can save you a lot of time.
-The styles injected into the DOM by MUI rely on class names that [follow a simple pattern](/styles/advanced/#class-names):
-`[hash]-Mui[Component name]-[name of the slot]`.
+As an example, let's say you want to change the `Slider` component's thumb from a circle to a square.
 
-⚠️ These class names can't be used as CSS selectors because they are unstable,
-however, MUI applies global class names using a consistent convention: `Mui[Component name]-[name of the slot]`.
+First, use your browser's dev tools to identify the class for the component slot you want to override:
 
-Let's go back to the above demo. How can you override the slider's thumb?
+<img src="/static/images/customization/dev-tools.png" alt="dev-tools" width="796" style="margin-bottom: 16px;" />
 
-<img src="/static/images/customization/dev-tools.png" alt="dev-tools" width="406" />
+Then, copy the target class of `MuiSlider-thumb`, place it inside the styles witihn the `sx` prop, and add your overrides.
```

Comment:
This reads strange, it doesn't feel like it's following the standard CSS terminology.

```suggestion
Then, use this class name (`MuiSlider-thumb`) to write a CSS selector within the `sx` prop (`& .MuiSlider-thumb`), and add your overrides inside.
```